cfgov/wellbeing/jinja2/wellbeing/results.html

```diff
@@ -113,12 +113,14 @@
             text-align: right;
         }
         .comparison-chart_list dd {
-            display: inline-block;
             width: 65%;
             border-bottom: 1px solid #b4b5b6;
             position: relative;
             bottom: 0.25em;
         }
+        .comparison-chart_list dd:not(.u-hidden) {
```

Comment:
Any concern about IE8 not supporting `:not`?

Reply:
IE8 won't get it because it's added by JS and IE8 isn't getting JS anyway.

Reply:
Realizing now this still wouldn't work. Added conditional `lt-i89` class to cover it.